src/@next/Icon/icons/ri-mail-line.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@roshnet can you help to remove height, width and fill here? 
do double check the steps here as well: https://aries.glints.design/?path=/story/next-icon--page#adding-new-icons

you'll know that you've saved the right file when it looks similar to the one from here: https://github.com/glints-dev/glints-aries/pull/839/files

Reply:
thanks @ninariccimarie, updated!